### PR TITLE
Add new tests and other improvements.

### PR DESCRIPTION
**Pull request by mezarin**
This PR:
- Adds Run tests action test run through the drop-down menu.
- Adds start/runTests/stop action tests run through the project pop-up menu. These tests stumbled upon knows issue : 272. They will be re-enabled when the issue is fixed.

### CODE REVIEW

File: src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java [hunk 1]
```diff
@@ -86,7 +86,97 @@ public void testDropDownStartAction() {
     }
 
     /**
-     * Prepares the the environment to run the tests.
+     * Tests Liberty Tools start/RunTests/stop actions run from the project's drop-down action menu.
+     */
+    @Test
+    public void testRunTestsActionUsingDropDownMenu() {
+        String testName = "testDropDownStartAction";
```
Comment (TrevCraw) [inline src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java:93]: update name for this test - maybe `testDropDownRunTests` ?

File: src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java [hunk 2]
```diff
@@ -86,7 +86,97 @@ public void testDropDownStartAction() {
...
+        TestUtils.validateLibertyServerStopped(testName, absoluteWLPPath);
+    }
+    
+    /**
+     * Tests Liberty Tools start/runTests/stop actions run from the project's pop-up action menu.
+     */
+    @Test
+    @Disabled("Until https://github.com/OpenLiberty/liberty-tools-intellij/issues/272 is fixed.")
+    public void testRunTestsActionUsingPopUpMenu() {
+        String testName = "testDropDownStartAction";
```
Comment (TrevCraw) [inline src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java:150]: update name for this test as well - maybe `testMenuRunTests` ?

File: src/test/java/io/openliberty/tools/intellij/it/TestUtils.java [hunk 3]
```diff
@@ -135,7 +135,7 @@ public static void validateAppStarted(String testName, String appUrl, String exp
         String msg = testName + ":validateAppStarted: Timed out while waiting for application under URL: " + appUrl + " to become available.";
         printTrace(TraceSevLevel.ERROR, msg);
         String wlpMsgLogPath = wlpInstallPath + "/wlp/usr/servers/defaultServer/logs/messages.log";
-        String msgHeader = "TESTCASE: " + testName;
+        String msgHeader = "Message lo for failed test: " + testName + ":validateAppStarted";
```
Comment (TrevCraw) [inline src/test/java/io/openliberty/tools/intellij/it/TestUtils.java:138]: message log ?

File: src/test/java/io/openliberty/tools/intellij/it/TestUtils.java [hunk 4]
```diff
@@ -179,10 +182,12 @@ public static void validateAppStopped(String testName, String appUrl, String wlp
         }
 
         // If we are here, the expected outcome was not found. Print the Liberty server's messages.log and fail.
+        String msg = testName + ":validateAppStopped: Timed out while waiting for application under URL: " + appUrl + " to stop.";
+        printTrace(TraceSevLevel.ERROR, msg);
         String wlpMsgLogPath = wlpInstallPath + "/wlp/usr/servers/defaultServer/logs/messages.log";
-        String msgHeader = "TESTCASE: " + testName;
+        String msgHeader = "Message lo for failed test: " + testName + ":validateAppStopped";
```
Comment (TrevCraw) [inline src/test/java/io/openliberty/tools/intellij/it/TestUtils.java:188]: message log ?

File: src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java (hunk 1 above)
Comment (mezarin) [reply src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java]: Thanks Trevor, fixed this one and the other comments.

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**